package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@skyux/navbar",
+  "version": "3.0.0",
+  "description": "SKY UX Navbar",
+  "main": "bundles/bundle.umd.js",
+  "module": "index.ts",
+  "engines": {
+    "node": ">=10.15.1"
+  },
+  "scripts": {
+    "build": "skyux build-public-library",
+    "skyux": "skyux",
+    "start": "skyux serve --hmr",
+    "test": "skyux test --coverage library --logFormat none",
+    "test:visual": "skyux e2e --logFormat none"
+  },
+  "keywords": [],
+  "author": "Blackbaud",
+  "license": "ISC",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/blackbaud/skyux-navbar.git"
+  },
+  "bugs": {
+    "url": "https://github.com/blackbaud/skyux-navbar/issues"
+  },
+  "homepage": "https://github.com/blackbaud/skyux-navbar#readme",
+  "peerDependencies": {
+    "@angular/common": "^4.3.6",
+    "@angular/core": "^4.3.6",
+    "@angular/http": "^4.3.6",
+    "@angular/platform-browser": "^4.3.6",
+    "@skyux/i18n": "^3.5.0",
+    "@skyux/popovers": "^3.0.0",
+    "rxjs": "5.4.3"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@angular/animations": "4.3.6",
+    "@angular/common": "4.3.6",
+    "@angular/compiler": "4.3.6",
+    "@angular/compiler-cli": "4.3.6",
+    "@angular/core": "4.3.6",
+    "@angular/forms": "4.3.6",
+    "@angular/http": "4.3.6",
+    "@angular/platform-browser": "4.3.6",
+    "@angular/platform-browser-dynamic": "4.3.6",
+    "@angular/router": "4.3.6",
+    "@blackbaud/skyux": "2.40.0",
+    "@blackbaud/skyux-builder": "1.31.1",
```

Comment:
Since we're referencing old Builder and SKY UX 2, listing out the individual component libraries might cause problems. I would remove all `@angular` and `@skyux` references, as well as the other non-Blackbaud dependencies (`core-js`, `rxjs`, etc.).